src/components/HistoryEventDetails/layouts/TokenExchangeEventDetails.js

```diff
@@ -43,15 +43,17 @@ import { spacing } from 'utils/variables';
 
 // Types
 import type { TokenExchangeEvent } from 'models/History';
+import type { Chain } from 'models/Chain';
 
 // Local
 import BaseEventDetails from './BaseEventDetails';
 
 type Props = {|
   event: TokenExchangeEvent,
+  chain: Chain,
 |};
 
-function TokenExchangeEventDetails({ event }: Props) {
+function TokenExchangeEventDetails({ event, chain }: Props) {
```

Comment:
Looking at these repeated changes, it would probably make sense put chain info in the `Event` type directly. wdyt?